orchestrator/tests/test_workspace_model_seeding.py

```
@@ -0,0 +1,177 @@
+
+def test_unknown_vendor_passes_through():
+    f = _factory()
+    assert f._openrouter_model_id("huggingface", "some-model") == "some-model"
+
+
+# --------------------------------------------------------------------------- #
+# seed_workspace_models — selection, idempotency, primary (fake session)
+# --------------------------------------------------------------------------- #
+
+class _Model:
+    def __init__(self, id, model_id, default=False, featured=False, pop=0):
```

Ruff (0.16.2)

[error] Function argument `id` is shadowing a Python builtin

(A002)